### PR TITLE
Bodily fluid now carries forensic data

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Bodily fluid stains now have the DNA of whodunnit.
image
Thank you and curse you for inspiring me

## How This Contributes To The Skyrat Roleplay Experience
It really doesn't

## Changelog

:cl:
fix: Bodily fluid now carries DNA of the person who created it
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
Quality of life.

**Comment by YakumoChen** [comment]
do we really WANT to know? I feel like this prebreaks security, rape etc has already been removed from security duties / space law.

File: modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm [hunk 1]
```diff
@@ -1235,4 +1192,12 @@ GLOBAL_LIST_INIT(strippable_human_erp_items, create_erp_strippable_list(list(
 	client.mob.hud_used.persistent_inventory_update(client.mob)
 
 /obj/item/proc/is_in_genital(mob/living/carbon/human/the_guy)
-	return !!(src == the_guy.penis || src == the_guy.vagina || src == the_guy.anus || src == the_guy.nipples)
+	return (src == the_guy.penis || src == the_guy.vagina || src == the_guy.anus || src == the_guy.nipples)
+
+/mob/living/proc/add_cum_splatter_floor(turf/the_turf)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm]: Documentation here please :)

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Overall I'm really indifferent to this being a thing or not, I guess it makes sense, worst case scenario we can just take it out if it causes problems, I guess.

**Comment by Iamgoofball** [comment]
this should not have player facing DNA because if a detective is having to scan this then something is already wrong

**Review by GoldenAlpharex — APPROVED** [review]
Looks good enough, yeah.

**Pull request merged**